src/pages/ServerRules.js

```diff
@@ -54,11 +53,20 @@ class ServerRules extends Component {
 								soliciting, and/or buying anything through our services is not
 								allowed.
 							</li>
-							<li>Discrimination of any kind will not be tolerated.</li>
+							<li>
+								Discrimination of any kind will not be tolerated. This includes discrimination of age, 
```

Comment:
I disagree with "discrimination of age". If Player A calls Player B "shut up 8 year old", does this mean we should punish him for 2 weeks for discrimination? Discrimination of age makes no sense imo. I suggest removing that part.

Reply:
Well, that situation would likely be considered disrespect or toxicity. Discrimination of age would mean that someone is being treated less favorably based on their age, while ageism is not a top concern on Warzone I do think that some people may dislike being criticized, excluded, or told that they aren't good enough, purely based on their age.

I think discrimination of age can typically be considered as toxicity at most, however we can make a change to it if a Senior Moderator or an Administrator shares their perspective.

Reply:
"Discrimination of age would mean that someone is being treated less favorably based on their age"

Please give me an example scenario of where discrimination of age happens on Warzone. I don't feel like there are any, so why should we even include it in the rules, as if it's some kind of important thing? People will only be confused about this rule and interpret it wrongly.

Reply:
Agreed, I'll go ahead and remove this as I no longer think it's necessary.